CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/providers/EsriMapProvider.java

```diff
@@ -21,20 +21,23 @@
 
 /**
  * A tiled base map being served by an ArcGIS Server. This class will ensure
- * tiles are exported in the Web Mercator projection with the correct alignment.
+ * tiles are exported in the Web Mercator projection with the correct alignment
+ * by exporting tiles as images.
  *
  * @author cygnus_x-1
  */
-public abstract class ArcgisMapProvider extends MapProvider {
+public abstract class EsriMapProvider extends MapProvider {
 
-    @Override
-    public int tileWidth() {
-        return 256;
+    public static enum MapServerType {
+        TILE,
```

Comment:
It would be great to have a comment about what `TILE` and `EXPORT` types mean.

Reply:
Done.